### PR TITLE
feat: change stats renderless component

### PR DESCRIPTION
**Pull request by JaimeTorrealba**

### CODE REVIEW

File: docs/guide/misc/stats.md [hunk 1]
```diff
@@ -7,14 +7,21 @@
 - MB MBytes of allocated memory. (Run Chrome with --enable-precise-memory-info)
 - CUSTOM User-defined panel support.
```
Comment (andretchen0) [inline docs/guide/misc/stats.md:8]: Maybe link to the [custom panel example](https://github.com/mrdoob/stats.js/blob/master/examples/custom.html) in the original repo here?

Or maybe provide a demo?

(I'm not sure how much time you want to spend here. Just a suggestion.)

**Review by andretchen0 — CHANGES_REQUESTED** [review]
Looks good!

It seems like everything is working well. ;) 

I made some suggestions for changes to wording, links, etc.

File: docs/guide/misc/stats.md (hunk 1 above)
Comment (JaimeTorrealba) [reply docs/guide/misc/stats.md:8]: I have never created a custom panel, so I'll just put the link to the repo here

File: docs/guide/misc/stats.md [hunk 2]
```diff
@@ -2,23 +2,30 @@
...
+| :------------ | -------- | ------- | ------------ |
+| **showPanel** | `number` | `0`     | FPS monitor. |
 
-| Name          | Type     | Default | Description          |
-| :------------ | -------- | ------- | -------------------- |
-| **showPanel** | `number` | `0`     | The initial monitor. |
+- 0: FPS Frames rendered in the last second. The higher the number the better.
+- 1: MS Milliseconds needed to render a frame. The lower the number the better.
+- 2: MB MBytes of allocated memory. (Run Chrome with --enable-precise-memory-info)
+- 3+: CUSTOM User-defined panel support, for mor info please check [stats.js](https://github.com/mrdoob/stats.js/).
```
Comment (andretchen0) [inline docs/guide/misc/stats.md:31]: > 3+: CUSTOM User-defined panel support, for mor info please check ...

3+: CUSTOM User-defined panel support. For more info please check ...

* 'support. For' – the other entries have periods, so just for consistency 
* 'mor' -> 'more'

**Review by andretchen0 — CHANGES_REQUESTED** [review]
There's just one change and then it's ready to go.

File: docs/guide/misc/stats.md (hunk 2 above)
Comment (JaimeTorrealba) [reply docs/guide/misc/stats.md]: Thanks for this

**Review by andretchen0 — APPROVED** [review]
👍

**Pull request merged**